utils.py

```diff
@@ -97,3 +99,92 @@ def get_details_from_id(alert_id):
 
     return retVal
 
+def parse_details_to_file_bug(details, oldest_alert):
+    branch = details['branch']
+    test = details['test']
+    platform = details['platform']
+    percent = details['percent']
+    push_date = details['push_date']
+    summary = ''
+
+    #add regression values
+    min_percent = min(percent)
+    max_percent = max(percent)
+    min_percent = re.findall('(?P<number>\d+)', min_percent)[0]
+    max_percent = re.findall('(?P<number>\d+)', max_percent)[0]
+    summary = summary + '%s-%s' %(min_percent, max_percent) + '% '
+
+    #add platform
+    flag_linux = 0
+    flag_win = 0
+    flag_mac = 0
+    flag_android = 0
+    for p in platform:
+        if 'Ubuntu' in p:
+            flag_linux = 1
+        if 'WIN' in p:
+            flag_win = 1
+        if 'Mac' in p:
+            flag_mac = 1
+        if 'Android' in p:
+            flag_android = 1
```

Comment:
I would make a json object:
if 'ubuntu 12.04 x64' in p:
  platforms[linux64] = 1
elif 'ubuntu' in p:
  platforms[linux32] = 1
...

then we could do something like this:
if platforms[linux32] and platforms[linux64]:
   add = add + "linux*"